docs_user/modules/proc_migrating-mon-from-controller-nodes-prerequisites.adoc

```diff
@@ -0,0 +1,158 @@
+.Prerequisites
```

Comment:
```suggestion
[id="Configuring-target-nodes-for-ceph-monitor-migration_{context}"]

= Configuring target nodes for Ceph Monitor migration
```

Reply:
Please confirm if you agree with this title. It seems like the procedure is describing how to prepare the target nodes for the Ceph Monitor migration.

Reply:
Yes, I agree, this is like having preliminary steps that are required to let the rest of the procedure work as expected.
I'm going to incorporate your comments first and push a new PS we can review.

Reply:
Done